pure-pwsh/async.ps1

```diff
@@ -22,6 +24,7 @@ $Script:OnIdleCallback = {
     $isTest ? (&prompt) : [Microsoft.PowerShell.PSConsoleReadLine]::InvokePrompt()
   }
   $pure._state.isPending = $false
+  Remove-Job $Script:UpdateJob
```

Comment:
Should we `-ErrorAction Ignore` this? Just in case the job has already been removed by some other process or race condition?